Vizzuality/half-earth-v3
# Add pause and mute. Tweak styles

## Add pause and mute. Tweak styles
### Description

- Add mute button and functionality
- Add pause button when the user hovers the waves and functionality

### Designs
https://www.figma.com/file/8lZsKUaGM6K9JOIG0orY76/HE---Design-System?node-id=2216%3A13471https://www.figma.com/file/8lZsKUaGM6K9JOIG0orY76/HE---Design-System?node-id=2216%3A13471

### Testing instructions
Hover the waves, you should see a pause button and be able to click it to pause the onboarding
You should be able to click the mute button to mute the recording even if the onboarding keeps going on

### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-329?atlOrigin=eyJpIjoiOGVhMDA4NTZlZjdmNGNmMDk3NDhjM2I4YTMwNmZlMzEiLCJwIjoiaiJ9
